### ISSUE TITLE
[IOS-XE] "show ip rpf" parsing failing when the RPF interface is  LISP or RPF type: unicast()

### ISSUE THREAD

**Issue by desiredrive** [CiscoTestAutomation/genieparser#903]
Genie exception with the following command:

"show ip rpf (ip)" or "show ip rpf vrf (vrf) (ip)"

Parser works with outputs like this:

```
show ip rpf 172.19.1.66
RPF information for ? (172.19.1.66)
  RPF interface: TenGigabitEthernet1/0/3
  RPF neighbor: ? (172.19.2.6)
  RPF route/mask: 172.19.1.66/32
  RPF type: unicast (isis)
  Doing distance-preferred lookups across tables
  Multicast Multipath enabled.
  RPF topology: ipv4 multicast base, originated from ipv4 unicast base
```

But Genie exception with the following:

```
show ip rpf vrf Campus 1.1.1.1
RPF information for ? (1.1.1.1)
  RPF interface: LISP0.4099
  RPF neighbor: ? (172.19.1.64)
  RPF route/mask: 0.0.0.0/5
  RPF type: unicast ()
  Doing distance-preferred lookups across tables
  Multicast Multipath enabled.
  RPF topology: ipv4 multicast base
```

Seen in Catalyst C9300 IOS-XE 17.12.3

**Comment by SohanTirpude** [comment]
Hello @desiredrive,

For this, I was able to reproduce the issue and found the possible solution as well. With the solution, the output now looks like this:
```
{
    "vrf": {
        "Campus": {
            "source_address": "1.1.1.1",
            "source_host": "?",
            "path": {
                "172.19.1.64 LISP0.4099": {
                    "interface_name": "LISP0.4099",
                    "neighbor_host": "?",
                    "neighbor_address": "172.19.1.64",
                    "route_mask": "0.0.0.0/5",
                    "table_type": "unicast",
                    "distance_preferred_lookup": True,
                    "lookup_topology": "ipv4 multicast base"
                }
            }
        }
    }
}
```
Can you please check and confirm if everything is correct so that I can raise the PR?

Thank you.

**Comment by desiredrive** [comment]
Hi @SohanTirpude , it looks good, I'd say it is what I expected. Thanks!

**Comment by SohanTirpude** [comment]
Hello @desiredrive,

Thank you for the confirmation. I have raised the PR and once it is merged, I will update you back here.

Thank you.

**Comment by SohanTirpude** [comment]
Hello @desiredrive,

The PR got merged and the fix will be available in the upcoming pyats 24.10 release. Hence, I am closing this ticket for now. If you are facing any other issues, kindly open a new ticket.

Thank you.
